### ISSUE TITLE
Any plans on prospective analyses?

### ISSUE THREAD

**Issue by jbracher** [epiforecasts/bp.delta#52]
It is likely that we will observe more transitions from one dominant variant to another. Is there a point in stating that the "best" model identified retrospectively here will be applied prospectively the next time this occurs in the countries we considered? If we post this as a preprint first or go with Wellcome Open there could be a first version laying out the retrospective analysis and specification of prospective analysis and we'd update it with a new version once we will have conducted it. Maybe that's a bit too experimental, but in principle I'd find that interesting.

**Comment by seabbs** [comment]
Yes, I think this is a great idea but perhaps for a second follow up piece of work vs the second version of this study? I think much of the work here is going to be with simulated scenarios based on real data and so there could just be to much going on. 

I am writing the code with an eye to future reuse with these kinds of questions in mind. For now lets add to the "future work" and hope get some results before a new variant of concern starts kicking off.
